migrations/message_store_postgres/content_script_version_2.nim

```diff
@@ -0,0 +1,68 @@
+const ContentScriptVersion_2* = """
+ALTER TABLE messages RENAME TO messages_backup;
+ALTER TABLE messages_backup DROP CONSTRAINT messageIndex;
```

Comment:
Yes, this is needed because we create it in https://github.com/waku-org/nwaku/blob/d1ac0669af332ddbdd10bf2c91818974ee2fe569/migrations/message_store_postgres/content_script_version_1.nim#L11

And when transitioning to version 2 we will create the new constraint in: https://github.com/waku-org/nwaku/blob/d1ac0669af332ddbdd10bf2c91818974ee2fe569/migrations/message_store_postgres/content_script_version_2.nim#L14